cpp/src/io/csv/reader_impl.cu

```diff
@@ -196,15 +193,10 @@ void erase_except_last(C& container, rmm::cuda_stream_view stream)
   container.resize(1, stream);
 }
 
-size_t find_first_row_start(char row_terminator, host_span<char const> data)
+constexpr std::array<uint8_t, 3> UTF8_BOM = {0xEF, 0xBB, 0xBF};
```

Comment:
TIL CSV BOM kthxby.

Reply:
Come to that:  Does UTF-16 have a different BOM?  We don't support it, do we?  Should we be detecting it and bailing?

Reply:
Apparently it does, (U+FEFF)! I don't think we need to take this into account, since we only support UTF-8, right?